containers/EmpSponsors.ts

```diff
@@ -28,6 +28,11 @@ interface PositionQuery {
   sponsor: { id: string };
   tokensOutstanding: string;
   collateral: string;
+  pendingWithdraw: string;
+  pendingTransfer: string;
+  withdrawalRequestPassTimestamp: string;
+  withdrawalRequestAmount: string;
```

Comment:
```suggestion
```

Reply:
this one I will keep though because it is used & returned from the query.